src/NosCore.Packets/ClientPackets/ExcListPacket.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NosCore.Core.Serializing;
+
+namespace NosCore.Packets.ClientPackets
+{
+    [PacketHeader("exc_list")]
+    public class ExcListPacket : PacketDefinition
+    {
+        [PacketIndex(0)]
+        public int Unknown { get; set; }
```

Comment:
wondering if it's not the amount of subpackets

Reply:
Probably, I did not finish this